editor.planx.uk/src/ui/Banner.tsx

```diff
@@ -49,8 +51,8 @@ function Banner(props: BannerProps) {
       color={props.color && props.color.text}
       display="flex"
       justifyContent="center"
-      textAlign="center"
-      px={2}
+      textAlign={isUsingAltTheme ? "left" : "center"}
+      px={isUsingAltTheme ? 0 : 2}
```

Comment:
Question: This pattern could get pretty annoying / noisy across the codebase. Do we just want to make this style change?

Reply:
:heavy_plus_sign: am pro just making this style change - agree this is noisy to introduce, and will be equally noisy to remove later on